src/osd/OSD.cc

```diff
@@ -4081,7 +4081,7 @@ void OSD::final_init()
 
   r = admin_socket->register_command("compact",
 				     asok_hook,
-				     "Commpact object store's omap."
+				     "Compact object store's omap."
```

Comment:
Inclined to accept, I don't expect anything to depend on a usage command description.